gitlabber/gitlab_tree.py

```diff
@@ -109,7 +110,11 @@ def get_subgroups(self, group, parent):
         subgroups = group.subgroups.list(as_list=False, archived=self.archived)
         self.progress.update_progress_length(len(subgroups))
         for subgroup_def in subgroups:
-            subgroup = self.gitlab.groups.get(subgroup_def.id)
+            try:
+                subgroup = self.gitlab.groups.get(subgroup_def.id)
+            except GitlabGetError as error:
+                log.warning(f"Group {subgroup.name} (ID: {subgroup.id}) retrieval error: {error}")
```

Comment:
can you change this to an error it's not a warning if you failed to clone a subgroup

Reply:
I think `GitlabGetError` is too wide this PR is trying to specifically deal with empty groups (404) can you add a check for that specifically and include that in the error message?

Reply:
Hey @ezbz sorry for the delay! I'm unsure why I don't get emails from my own PRs :/

I've narrowed down the error with the response code and added a more descriptive message if that happens telling the user to check for permissions issues.